platform/ios/src/MGLMapboxEvents.m

```diff
@@ -665,11 +656,13 @@ - (void)locationManager:(MGLLocationManager *)locationManager didUpdateLocations
         double accuracy = 10000000;
         double lat = floor(loc.coordinate.latitude * accuracy) / accuracy;
         double lng = floor(loc.coordinate.longitude * accuracy) / accuracy;
+        double horizontalAccuracy = loc.horizontalAccuracy;
```

Comment:
By the way, technically this should be a CLLocationAccuracy, which happens to be a typedef for double.